### PR TITLE
(UPDATED) Add prof features

### PR DESCRIPTION
**Pull request by berserker1**
I will handle the process of creating prof page in a later Pull Request

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sys.path.append(PATH_TO_MODULE)
+if(os.path.exists(PATH_TO_MODULE)):
+    print('You have pywikibot repo downloaded')
+
+else:
+    print('Downloading pywikibot github repository')
+    Repo.clone_from('https://gerrit.wikimedia.org/r/pywikibot/core.git', 'wiki_files')
+    print("Downloaded pywikibot code from Github into the folder named wiki files")
+
+    link=input('Write the link for the wikipage : ')
```
Comment (Ayushk4) [inline setup.py]: Leave space before and after the assignment operator

Comment (berserker1) [reply setup.py]: why is that required

Comment (Ayushk4) [reply setup.py]: It is the convention used for better code readability. Look up pep8speaks.

**Comment by Ayushk4** [comment]
LGTM :+1:

**Pull request merged**